.gitignore

```diff
@@ -9,7 +9,7 @@
 .buildlog/
 .history
 .svn/
-lib/config/SwitchConfig.dart
+lib/config/
```

Comment:
Let's remove this again since it does not prevent commits of those files. Also the other files in that folder should not be ignored (`PEBRAConfig.dart` and `SharedPreferencesConfig.dart`).